test/values.test.js

```diff
@@ -255,6 +255,37 @@ describe('Values', () => {
     )
   })
 
+  test('pretty print Match', () => {
+    assertPrint(
+      new Query(q.Lambda('_', q.Match(q.Index('idx')))),
+      'Query(Lambda("_", Match(Index("idx"))))'
+    )
+
+    assertPrint(
+      new Query(q.Lambda('_', q.Match(q.Index('idx'), 'str'))),
+      'Query(Lambda("_", Match(Index("idx"), "str")))'
+    )
+
+    assertPrint(
+      new Query(q.Lambda('_', q.Match(q.Index('idx'), 'str', 10))),
+      'Query(Lambda("_", Match(Index("idx"), "str", 10)))'
```

Comment:
Please add a other version of that assertion that pass the arguments to Match as an array

Reply:
Will do!

Reply:
Done!